experiment/simulation/css/main.css

```diff
@@ -1 +1,228 @@
-/* You CSS goes in here */
\ No newline at end of file
+.main{
+    display: flex; 
+    justify-content: center; 
+    align-items: center;
+}
+
+.title {
+    text-align: center;
+    margin-top: 5px;
+    font-size: 2.5vw;
+    z-index: -1;
+}
+
+.nav-link ,.navbar-nav,.navbar ,.navbar-custom{
```

Comment:
It'll be better to move the background part into its separate class  like nav-bg. For padding you can use bootstrap built in padding classes https://getbootstrap.com/docs/4.0/utilities/spacing/